defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+podman_user_name: osbs-podman
+podman_user_uid: 2022
```

Comment:
Since we need to know the uid while adding podman remote connections ( path to the podman socket `/run/user/<uid>/podman/podman.sock` ), the uid should be set explicitly. uid starts from 1000 might not be good, I choose this special number, stands for the year, and OSBS2.0 (20) ssh user (22).
welcome for any suggestions as well.